### PR TITLE
Update apx' manpage.md

### PR DESCRIPTION
**Pull request by JMarcosHP**

### CODE REVIEW

File: docs/apx/manpage.md [hunk 1]
```diff
@@ -175,36 +188,37 @@ Examples:
...
 ```
 
-## REMOVE
+
+## RUN
 
 ```markdown
-Description:
-    Remove packages inside a managed container.
+Description:
```
Comment (kbdharun) [inline docs/apx/manpage.md:195]:
```suggestion
Description:
```

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
Add Closes #52 to this PRs description.

**Comment by JMarcosHP** [comment]
> Add Closes #52 to this PRs description.

Committed

**Comment by mirkobrombin** [comment]
The log command is actually unimplemented and should be removed from the help message until it is

**Comment by kbdharun** [comment]
> The log command is actually unimplemented and should be removed from the help message until it is

Then I will update the manpage too then. I added it from here.

**Comment by kbdharun** [comment]
Thanks for your contribution, I am closing this PR as Apx's behaviour has changed in v2 and we will update the overall page soon.

**Pull request closed without merging**